src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs

```diff
@@ -37,6 +38,10 @@ public class NewAutomationUpdateManagementAzureQuery : AzureAutomationBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
         public string[] Locaton { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
+        [LocationCompleter("Microsoft.Automation/SoftwareUpdateConfiguration")]
+        public string[] Location { get; set; }
```

Comment:
Simply add an `[Alias("Locaton")]` to `Location` and remove the `Locaton` property.

Reply:
Thank you @MiYanni , this is more simpler